Formula/bzt.rb

```diff
@@ -166,9 +166,7 @@ class Bzt < Formula
   end
 
   def install
-    # Fix "ld: file not found: /usr/lib/system/libsystem_darwin.dylib" for lxml
```

Comment:
Why remove that comment?

Reply:
Just for cleaning reason. By way, I return it back